active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,349 @@
+
+![A prototype of five dots, the PipsPager, laid out horizontally underneath a FlipView showing a picture of a bicycle. The third dot in the PipsPager is selected indicated the 3rd page of content.](images/PipsPager_FlipViewExample.png)
+
+XAML
+``` XAML
+<StackPanel>
+  <FlipView x:Name="Gallery" MaxWidth="400" Height="270" ItemsSource="{x:Bind Pictures}">
+      <FlipView.ItemTemplate>
+          <DataTemplate x:DataType="x:String">
+              <Image Source="{x:Bind Mode=OneWay}"/>
+          </DataTemplate>
+      </FlipView.ItemTemplate>
+  </FlipView>
+
+  <!-- The SelectedPageIndex is bound to the FlipView to keep the two in sync -->
+  <muxc:PipsPager x:Name="FlipViewPipsPager"
+      HorizontalAlignment="Center"
+      Margin="0, 10, 0, 0"
+      NumberOfPages="{x:Bind Pictures.Count}"
+      SelectedPageIndex="{x:Bind Path=Gallery.SelectedIndex, Mode=TwoWay}" />
+</StackPanel>
+```
+
+#### Pip and Button customization
+
+The navigational buttons and pips are able to be customized via the PreviousButtonStyle, NextButtonStyle, SelectedPipStyle, and DefaultPipStyle.
+
+If the PreviousButtonStyle or NextButtonStyle sets the Visibility property on the navigational button, it will take precedence over PreviousButtonVisibility or NextButtonVisibility respectively unless they are set to the PipsPagerButtonVisibility value, "Collapsed". 
+If the PreviousButtonVisibility or NextButtonVisibility is set to "Collapsed", the styles set will not overwrite the Visibility of the button itself.
+
+Similarly, the SelectedPipStyle and DefaultPipStyle will take precedence over the values in the theme resources.
+
+![A prototype of five dots, the PipsPager, laid out horizontally with caret-style navigational arrows](images/PipsPager_CustomNavButtons.png)
+
+XAML
+``` XAML
+<Page.Resources>
+    <Style x:Key="NavButtonBaseStyle" TargetType="Button" BasedOn="{StaticResource PipsPagerNavigatonButtonBaseStyle}">
+        <Setter Property="Width" Value="30" />
+        <Setter Property="Height" Value="30" />
+        <Setter Property="FontSize" Value="12" />
+    </Style>
+
+    <Style x:Key="PreviousButtonStyle" BasedOn="{StaticResource NavButtonBaseStyle}" TargetType="Button">
+        <Setter Property="Content" Value="&#xEDDB;" />
+    </Style>
+
+    <Style x:Key="NextButtonStyle" BasedOn="{StaticResource NavButtonBaseStyle}" TargetType="Button">
+        <Setter Property="Content" Value="&#xEDDC;" />
+    </Style>
+</Page.Resources>
+
+<muxc:PipsPager x:Name="CustomNavButtonPipsPager"
+  PreviousButtonStyle="{StaticResource PreviousButtonStyle}"
+  NextButtonStyle="{StaticResource NextButtonStyle}"
+  PreviousButtonVisibility="VisibleOnPointerOver" 
+  NextButtonVisibility="VisibleOnPointerOver" />
+```
+## PipsPager member notes
+
+| Name | Description| Default | 
+|:---:|:---|:---|
+| NumberOfPages | Sets the max number of pages the index control will iterate through. The default will represent an infinite page range. | -1
```

Comment:
What does infinite mean? Is there a special UX?

Reply:
@MikeHillberg  Infinite mean that the number of pages is literally infinite and the pager will scroll through the displayed pips like a scroll viewer. We’ll attach the gifs to display behaviour

Reply:
@MikeHillberg would you recommend dedicating an example in the docs to infinite pips?

Reply:
Recommend: just a line of text would be great.